src/Symfony/Component/Form/Extension/Validator/Event/PostValidateEvent.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Extension\Validator\Event;
+
+use Symfony\Component\Form\FormEvent;
+
+/**
+ * This event is dispatched after validation completes.
```

Comment:
maybe avoid duplicating the doc between here and the `Events.php`?
what about adding and `@see ...` instead ?

or viceversa, the full doc here and on the event class just the reference

Reply:
Good idea, will do.